src/java_bytecode/java_bytecode_convert_class.cpp

```diff
@@ -27,11 +27,13 @@ class java_bytecode_convert_classt:public messaget
     symbol_tablet &_symbol_table,
     message_handlert &_message_handler,
     bool _disable_runtime_checks,
-    size_t _max_array_length):
+    size_t _max_array_length,
```

Comment:
Being part does not mean that it has the same semantics: affecter fr. = assign engl.